internal/cmd/login.go

```diff
@@ -525,12 +600,16 @@ func promptSkipTLSVerify() error {
 // Returns the host address of the Infra server that user would like to log into
 func promptServer(options loginCmdOptions) (string, error) {
 	if options.NonInteractive {
-		return "", fmt.Errorf("Non-interactive login requires the [SERVER] argument")
+		return "", UserFacingError{
+			UserFacingMessage: "Non-interactive login requires the [SERVER] argument.",
+		}
 	}
 
 	config, err := readOrCreateClientConfig()
 	if err != nil {
-		return "", err
+		return "", UserFacingError{
+			Underlying: err,
+		}
```

Comment:
I think it would be nice to avoid having to wrap errors with `UserFacingError` if we aren't going to add a `UserFacingMessage`.

Instead of this wrapping , we could change `main()`. to use `Internal Error` instead of `Unhandled error`.

Reply:
will do!